src/main/java/nl/tudelft/pixelperfect/client/EventsMessage.java

```diff
@@ -0,0 +1,82 @@
+package nl.tudelft.pixelperfect.client;
+
+import com.jme3.network.AbstractMessage;
+import com.jme3.network.serializing.Serializable;
+
+
+/**
+ * Message type that sends the event to the client.
+ * 
+ * @author Dmitry Malarev
+ *
+ */
+@Serializable
+public class EventsMessage extends AbstractMessage {
+
+  public int id;
+  public String type;
+  public Long timestamp;
+  public Long duration;
+
+  /**
+   * Constructs a new EventsMessage instance.
+   * 
+   */
+  public EventsMessage() {
+  }
+
+  /**
+   * Changes the content of the message.

```

Comment:
A constructor does not "change" the values, it initializes them. I suggest describing the constructor method with something along the lines of "Constructs a new EventsMessage instance".